setup.py

```diff
@@ -23,8 +25,87 @@
           " install setuptools).", file=sys.stderr)
     sys.exit(1)
 
-sys.path.insert(0, os.path.abspath('lib'))
-from ansible.release import __version__, __author__
+
+def find_package_info(*file_paths):
+    # Open in Latin-1 so that we avoid encoding errors.
+    # Use codecs.open for Python 2 compatibility
+    try:
+        f = codecs.open(os.path.join(*file_paths), 'r', 'latin1')
+        info_file = f.read()
+        f.close()
```

Comment:
Any reason not to use a CM?
```suggestion
        with codecs.open(os.path.join(*file_paths), 'r', 'latin1') as f:
            info_file = f.read()
```

Reply:
Sorry, that was a copy/paste from pypa, and then slightly modified.